front-project/src/components/signInForm.jsx

```diff
@@ -35,16 +39,20 @@ export default function SignInForm() {
         <Form
           onSubmit={(values) => {
             const errors = {};
-            if (!values.email) {
-              errors.email = "Email required";
+            if (!values.username) {
+              errors.username = "username required";
             }
             if (!values.password) {
               errors.password = "Password required";
             }
             if (Object.keys(errors).length) {
               return errors;
             }
-            console.log(values);
+
+            if (Object.keys(errors).length === 0) {
```

Comment:
we don't need to check errors length here because if errors.length > 0 you exit the function in previous block